### PR TITLE
Fix Podcast Description None

### PR DESCRIPTION
**Pull request by Hiltronix**
Error occurs when saving changes to sqlite DB when desc is None, as SQL field "description" is set "NOT NULL" in table "podcasts".  Desc must be empty string not None.

### CODE REVIEW

**Comment by elelay** [comment]
Thanks for the PR!
Do you have an example of a youtube channel triggering the exception?

File: src/gpodder/util.py [hunk 1]
```diff
@@ -1009,7 +1009,10 @@ def get_first_line(s):
     Returns only the first line of a string, stripped so
     that it doesn't have whitespace before or after.
     """
-    return s.strip().split('\n')[0].strip()
+    try:
```
Comment (elelay) [inline src/gpodder/util.py]: Can you do it as this?
```
if s:
    return s.strip().split('\n')[0].strip()
return ''
```
so there is no blind `except:`

Comment (Hiltronix) [reply src/gpodder/util.py]: Updated.

**Comment by Hiltronix** [comment]
Hi Eric,

I believe the YouTube channel I was finding this issue with was cnxlinux:
https://www.youtube.com/user/cnxlinux

He doesn't have a description for his YT channel.

And the errors as shown in the log are:

2019-08-19 15:05:26,055 [gpodder.dbsqlite] ERROR: Cannot save <gpodder.gtkui.model.GPodcast object at 0x056fc5e0>: NOT NULL constraint failed: podcast.description
Traceback (most recent call last):
  File "C:\Program Files (x86)\gPodder\lib\python3.7\site-packages\gpodder\dbsqlite.py", line 220, in _save_object
    cur.execute(sql, values)
sqlite3.IntegrityError: NOT NULL constraint failed: podcast.description
2019-08-19 15:05:26,058 [gpodder.log] ERROR: Uncaught exception: Traceback (most recent call last):
  File "C:\Program Files (x86)\gPodder\lib\python3.7\site-packages\gpodder\gtkui\main.py", line 2395, in on_after_update
    self.update_podcast_list_model(select_url=url)
  File "C:\Program Files (x86)\gPodder\lib\python3.7\site-packages\gpodder\gtkui\main.py", line 2245, in update_podcast_list_model
    self.podcast_list_model.set_channels(self.db, self.config, self.channels)
  File "C:\Program Files (x86)\gPodder\lib\python3.7\site-packages\gpodder\gtkui\model.py", line 781, in set_channels
    self.update_by_iter(iter)
  File "C:\Program Files (x86)\gPodder\lib\python3.7\site-packages\gpodder\gtkui\model.py", line 873, in update_by_iter
    downloaded, unplayed)
  File "C:\Program Files (x86)\gPodder\lib\python3.7\site-packages\gpodder\gtkui\model.py", line 717, in _format_description
    description_markup = cgi.escape(util.get_first_line(channel.description) or ' ')
  File "C:\Program Files (x86)\gPodder\lib\python3.7\site-packages\gpodder\util.py", line 1012, in get_first_line
    return s.strip().split('\n')[0].strip()
AttributeError: 'NoneType' object has no attribute 'strip'

**Comment by elelay** [comment]
OK, thanks for the reference.
Merging now :+1:

**Pull request merged**